### PR TITLE
feat: update doc follow themes, update ink:theme classes

### PR DESCRIPTION
**Pull request by fran-ink**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,12 +34,58 @@ function App() {
...
+  ...
+}
+```
+
+To see examples on specific colors that you can override, check the following [theme](https://github.com/inkonchain/ink-kit/tree/main/src/styles/theme) section of the Ink Kit repository.
 
 ## Resources
 
-- **Documentation**: Browse components and examples on our [Storybook page](https://inkkit.inkonchain.com/)
-- **Contributing**: Open a PR right here on our [GitHub repository](https://github.com/inkonchain/ink-kit)
```
Comment (ink-victor) [inline README.md:51]: I would revert this change. `Browse components and examples in the sidebar` does not mean anything in the README.

**Review by ink-victor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fran-ink) [reply README.md:51]: Now wondering if there is some way to reuse some of the same content for both
